skimage/morphology/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""Utilities that process images based on shapes.
```

Comment:
I find there's a slight wording issue here, because it sounds as if *images* had to be "based on shapes" when, really, what is intended is that the actual *processing* is based on shapes (geometry).

Reply:
hmmm, I will improve the wordings. 
Thank you!